contracts/interfaces/IBasePool_v_1_1.sol

```diff
@@ -81,10 +81,9 @@ interface IBasePool_v_1_1 {
         uint256 liquidityBnd1,
         uint256 liquidityBnd2
     );
-
+    event LpWhitelistUpdate(address indexed maxLoanPerColl, bool isApproved);
```

Comment:
odd name for the address, but it technically works!

Reply:
good catch, fixed